### ISSUE TITLE
Add panel for configuring sort order of request

### ISSUE THREAD

**Issue by kevinhinterlong** [yavin-dev/framework#1286]
Do we have a way for users to control the sort order?

_Originally posted by @jkusa in https://github.com/yavin-dev/framework/pull/1280#discussion_r593545359_

**Comment by jkusa** [comment]
@kevinhinterlong is this issue still valid?

**Comment by kevinhinterlong** [comment]
Yep there's not way to easily change things like sort by browser then age
